### DISCUSSION TITLE
QC checklist/process for utility functions

### DISCUSSION THREAD

**Discussion by mattroumaya** [Gilead-Public/gsm#403] [General]
The [QC Checklist](https://github.com/Gilead-BioStats/gsm/wiki/QC-Checklist) is working well for standard mapping and assess functions, but might not be entirely applicable for utility functions. 

Should we create a separate "utility/ad-hoc" checklist, or borrow from the current QC Checklist and edit accordingly? 

Some of the utility functions could certainly use some stress testing and scrutiny during QC, but wanted to start a discussion to see if there are any good ways to standardize QC for non-standard functions.

**Comment by jwildfire** [comment]
Yeah, something less rigorous is definitely ok. Not sure i have a strong opinion on process (for once!). I wonder if some combination of doc review + test coverage would make sense? Also fine to just liberally edit the existing checklist on a case-by-case basis.
